2021-09-21 19:10:41,983 INFO    MainThread:25189 [wandb_setup.py:_flush():69] setting env: {}
2021-09-21 19:10:41,983 INFO    MainThread:25189 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-21 19:10:41,984 INFO    MainThread:25189 [wandb_init.py:_log_setup():348] Logging user logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_191041-338kedz6/logs/debug.log
2021-09-21 19:10:41,984 INFO    MainThread:25189 [wandb_init.py:_log_setup():349] Logging internal logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_191041-338kedz6/logs/debug-internal.log
2021-09-21 19:10:41,984 INFO    MainThread:25189 [wandb_init.py:_jupyter_setup():300] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f7b00573430>
2021-09-21 19:10:41,985 INFO    MainThread:25189 [wandb_init.py:init():381] calling init triggers
2021-09-21 19:10:41,985 INFO    MainThread:25189 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-21 19:10:41,985 INFO    MainThread:25189 [wandb_init.py:init():430] starting backend
2021-09-21 19:10:41,985 INFO    MainThread:25189 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-21 19:10:41,993 INFO    MainThread:25189 [backend.py:ensure_launched():135] starting backend process...
2021-09-21 19:10:41,999 INFO    MainThread:25189 [backend.py:ensure_launched():139] started backend process with pid: 25230
2021-09-21 19:10:42,001 INFO    MainThread:25189 [wandb_init.py:init():435] backend started and connected
2021-09-21 19:10:42,062 INFO    MainThread:25189 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-09-21 19:10:42,063 INFO    MainThread:25189 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-21 19:10:42,064 INFO    MainThread:25189 [wandb_init.py:init():494] updated telemetry
2021-09-21 19:10:42,065 INFO    MainThread:25189 [wandb_init.py:init():517] communicating current version
2021-09-21 19:10:43,017 INFO    MainThread:25189 [wandb_init.py:init():522] got version response 
2021-09-21 19:10:43,018 INFO    MainThread:25189 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-21 19:10:43,403 INFO    MainThread:25189 [wandb_init.py:init():557] starting run threads in backend
2021-09-21 19:10:44,502 INFO    MainThread:25189 [wandb_run.py:_console_start():1605] atexit reg
2021-09-21 19:10:44,503 INFO    MainThread:25189 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-09-21 19:10:44,503 INFO    MainThread:25189 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-09-21 19:10:44,504 INFO    MainThread:25189 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-21 19:10:44,505 INFO    MainThread:25189 [wandb_init.py:init():582] run started, returning control to user process
2021-09-21 19:10:44,508 INFO    MainThread:25189 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 19:10:44,508 INFO    MainThread:25189 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 19:10:44,730 INFO    MainThread:25189 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 19:10:44,731 INFO    MainThread:25189 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 19:10:44,731 INFO    MainThread:25189 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 19:10:44,990 INFO    MainThread:25189 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 19:10:45,843 INFO    MainThread:25189 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 19:10:45,843 INFO    MainThread:25189 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 19:10:45,992 INFO    MainThread:25189 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 19:10:45,993 INFO    MainThread:25189 [wandb_init.py:_pause_backend():262] pausing backend
2021-09-21 19:10:45,994 INFO    MainThread:25189 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 19:10:46,398 INFO    MainThread:25189 [wandb_init.py:_resume_backend():271] resuming backend
2021-09-21 19:10:46,409 INFO    MainThread:25189 [wandb_setup.py:_flush():69] setting env: {}
2021-09-21 19:10:46,409 INFO    MainThread:25189 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-21 19:10:46,409 INFO    MainThread:25189 [wandb_init.py:_log_setup():348] Logging user logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_191046-17ue97ot/logs/debug.log
2021-09-21 19:10:46,409 INFO    MainThread:25189 [wandb_init.py:_log_setup():349] Logging internal logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_191046-17ue97ot/logs/debug-internal.log
2021-09-21 19:10:46,410 INFO    MainThread:25189 [wandb_init.py:init():381] calling init triggers
2021-09-21 19:10:46,410 INFO    MainThread:25189 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-21 19:10:46,411 INFO    MainThread:25189 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 338kedz6
2021-09-21 19:10:46,414 INFO    MainThread:25189 [wandb_run.py:finish():1275] finishing run mlwell-rgb/gta/338kedz6
